docs/design/autoware-architecture/sensing/data-types/radar-data/radar-objects-data.md

```diff
@@ -0,0 +1,88 @@
+# Radar objects data pre-processing design
+
+## Overview
+
+### Pipeline
+
+This diagram describes the pre-process pipeline for radar objects.
+
+![radar-objects-sensing](image/radar-objects-sensing.drawio.svg)
+
+### Interface
+
+- Input
+  - Radar data from device
+  - Twist information of ego vehicle motion
+- Output
+  - Merged radar objects information
+
+### Note
+
+- The radar pre-process package filter noise through the `ros-perception/radar_msgs/msg/RadarTrack.msg` message type with sensor coordinate.
+- It is recommended to change the coordinate system from each sensor to base_link with message converter.
+- If there are multiple radar objects, the object merger package concatenate these objects.
+
+## Input
+
+Autoware uses radar objects data type as [radar_msgs/msg/RadarTracks.msg](https://github.com/ros-perception/radar_msgs/blob/ros2/msg/RadarTracks.msg).
+In detail, please see [Data message for radars](supported-functions/data-message.md).
+
+## Supported functions
```

Comment:
[IMO]
"Reference implementation" might be better?

Reply:
Fixed at
https://github.com/autowarefoundation/autoware-documentation/pull/518/commits/03ba9be04565ebc102a0619a59c265c89e5b4b38
https://github.com/autowarefoundation/autoware-documentation/pull/518/commits/fab964c17a773289122fb08ae59fbc5a87d5d0b5